ovis_robotiq_gripper/src/ovis_robotiq_gripper/__init__.py

```diff
@@ -0,0 +1 @@
+#autogenerated by ROS python message generators
```

Comment:
I'm not sure if this should be inside the git repo. What do you think? Is there's more info online? Because, if that's auto-generated, I'm assuming at build time, we don't need to keep it in the VCS.

Reply:
It is required according to https://docs.python.org/3/tutorial/modules.html#packages